src/django/contricleaner/tests/sector_cache_mock.py

```diff
@@ -1,4 +1,6 @@
-from contricleaner.lib.sector_cache_interface import SectorCacheInterface
+from contricleaner.lib.client_abstractions.sector_cache_interface import (
+    SectorCacheInterface
+)
 
 
 class MockSectorCache(SectorCacheInterface):
```

Comment:
I suppose it's suitable to rename the class for more explicit after renaming the file name.

Reply:
Will do.
Thanks👍

Reply:
Done